cypress/e2e/contactForm.cy.js

```diff
@@ -1,16 +1,16 @@
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
```

Comment:
There is a missing semicolon at the end of this import statement. In JavaScript, it's a good practice to end statements with a semicolon to avoid potential issues.